libs/net/services/Actions/IngestAction.cs

```diff
@@ -95,7 +95,7 @@ public IngestAction(IApiService api, IOptions<TOptions> options, ILogger<IngestA
         if (reference != null)
         {
             if (reference.Status != (int)status) reference.Status = (int)status;
-            reference = await this.Api.UpdateContentReferenceAsync(reference, Headers);
+            reference = await this.Api.UpdateContentReferenceAsync(reference);
```

Comment:
Removed header implementation and moved it to a lower level so that it's done in one place